ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+a remote call to one of the remote servers (`server2`,`server3`), which are receivers.
+The installation directory for the `server1` is referred as `__{jbossHomeName}_1__`,
+for the `server2` it's `__{jbossHomeName}_2__` and for the `server3` it's `__{jbossHomeName}_3__`.
+
+[source,sh,subs="+quotes,attributes+"]
+----
+# considering the ${jbossHomeName} is installation directory of the {productName}
+cp -r ${jbossHomeName} server1
+{jbossHomeName}_1="$PWD/server1"
+cp -r ${jbossHomeName} server2
+{jbossHomeName}_2="$PWD/server2"
+cp -r ${jbossHomeName} server3
+{jbossHomeName}_3="$PWD/server3"
+----
+
+[#_configure_the_credentials_for_remote_call_authentication]
+=== Configure the credentials for remote call authentication
+
+For the remote call could be processed successfully from `server1` to either `server2`
+or to `server3` there has to be created a user at the receiver side
+which the call will be authenticated with.
+
+Use the following procedures for `server2` and `server3`.
+Run the procedure for directories
+`__{jbossHomeName}_2__` and `__{jbossHomeName}_3__`.
+
+[#add_application_user]
+// Add the Authorized Application User
+include::../shared-doc/add-application-user.adoc[leveloffset=+3]
+
+[NOTE]
+====
+For the `add-user.sh` (or `.bat`) command you may add a parameter `-ds`.
+The command line attribute `-ds` ensures that after the user is added
+there is printed the secret value that will be used in setting up
+the remote outbound connection at `server1`.
+
+The `-ds` parameter outputs:
+
+[code,sh]
+----
+To represent the user add the following to the server-identities definition <secret value="cXVpY2tzdGFydFB3ZDEh" />
+----
+====
+
+The `server1` has to be configured.
+Take a look at script `${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/remote-configuration.cli`
+to understand the commands that will be executed.
+The `cli` script is configured with `cli.local.properties`
+to run in https://wildfly.org/news/2015/03/13/Offline-CLI/[embedded mode] against the `standalone.xml`.
+
+[[remote_configuration_cli]]
+The CLI script `remote-configuration.cli` configures several things in the application server.
+
+* It configures a `remote outbound socket` which points to the port where EJB remoting endpoint
+  can be reached at `server2`.
+* It configures https://wildscribe.github.io/WildFly/18.0/subsystem/remoting/remote-outbound-connection/index.html[`remote outbound connection`]
+  which is referenced in the war deployment for connecting to the remote server via EJB remoting
+  (see `${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/src/main/webapp/WEB-INF/jboss-ejb-client.xml`).
+* It defines the security realm which utilizes the credentials added in the section <<add_application_user, Add the Authorized Application User>>.
+* Then the deployment of `server1` refers with property `-Dwildfly.config.url` to the configuration file `custom-config.xml`
```

Comment:
```suggestion
* Then the deployment of `server1` is updated with the property `-Dwildfly.config.url` pointing to the configuration file `custom-config.xml`
```
If the property already exists, I suggest:

"Then, in the deployment of `server1`, the property `-Dwildfly.config.url` is updated to point to the configuration file `custom-config.xml`